docs/src/pages/observability_.mdx

```diff
@@ -0,0 +1,89 @@
+We recommend you set "`APPROZIUM_LOG_FORMAT=json`" and "`APPROZIUM_LOG_LEVEL=info`" when running Approzium. This will result
+in the following:
+
+- Logs will be outputted in a JSON format that's easily consumable by log viewing tools such as the ELK stack and DataDog.
+- The `request_id` field may easily be searched for a request's full lifecycle.
+- If the caller's identity for the `request_id` was confirmed, it will be logged.
+- Any malicious-looking activity that occurred in the request will be logged at the `WARN` level.
+
+## Metrics
+
+Metrics are exposed at the `/v1/metrics/prometheus` endpoint.
+
+### Performance-Centric Metrics
+
+These metrics help understand performance. The number of requests, responses, and error responses help describe the load
+Approzium is under. Total request milliseconds is an end-to-end metric regarding how long Approzium handles a request.
+The other request milliseconds are for helping to understand where Approzium's request time is being spent. We do execute
+some requests in parallel to minimize request time, so the sum of sub-request time may exceed total request time.
+
+The AWS request being executed is the `get_caller_identity` call. The password request is where we call your configured
+back-end (HashiCorp Vault, for example) to retrieve passwords that will be used to build database connections.
+
+Total requests and total error responses can be considered significant to security because if an attacker is attempting
+to [DOS](https://en.wikipedia.org/wiki/Denial-of-service_attack) or [fuzz](https://owasp.org/www-community/Fuzzing)
+Approzium, these will rise at an unexpectedly high level.
+
+| METRIC                                          | SECURITY | TYPE    | DESCRIPTION                                                                             |
+|-------------------------------------------------|----------|---------|-----------------------------------------------------------------------------------------|
+| approzium_total_requests                        | yes      | counter | Total number of requests                                                                |
+| approzium_total_responses                       | no       | counter | Total number of responses                                                               |
+| approzium_total_error_responses                 | yes      | counter | Total number of error responses                                                         |
+| approzium_total_request_milliseconds            | no       | gauge   | Total request milliseconds                                                              |
+| approzium_total_aws_request_milliseconds        | no       | gauge   | Total AWS request milliseconds                                                          |
+| approzium_total_password_request_milliseconds   | no       | gauge   | Total password retrieval milliseconds                                                   |
```

Comment:
Great work! I am a total noob with metrics - what does the "security" column here mean?

Reply:
Oh, that's valuable feedback. I intended for it to make it easy to tell if it's a security-related metric. I'll add more explanation to the column name so that's more apparent.

Reply:
Hopefully this helps: https://github.com/approzium/approzium/pull/95/commits/75c759a682f619bab585f8f2d4ee19218a5ca2e3